docs/_pages/release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Release"
+permalink: /release/
+toc: false
+---
+
+The following steps must be executed to release a new version of Phoenicis:
+* Create release branches (e.g. "5.0-alpha") for phoenicis and scripts
```

Comment:
Do we really want to use branches for the releases?
Normally you use the `Releases` tab (together with git tags) on GitHub to manage releases.

Reply:
We will use the GitHub Release tab. You can trigger releases for branches as well. The big advantage is that you can have bugfix releases in the release branch and continue development in master.

[Here](https://datasift.github.io/gitflow/IntroducingGitFlow.html) is a description of GitFlow. As we do not have a development branch (would be a bit difficult with forking etc), you should read that description as if our master branch is the development branch. 
Now the question: Which branch shall be used for releases? Obviously, the master branch cannot be used because it is used as development branch and therefore receives changes rather often which can possibly break it. Moreover, the release branch may receive only bugfixes which also rules out master.
So the best and simplest approach we can use in my opinion is to create release branches, merge bug fixes to the release branch and to master and use the release branch to trigger a GitHub release.